pyuvsim/tests/test_simsetup.py

```diff
@@ -362,6 +381,15 @@ def test_freq_time_params():
     ttest = pyuvsim.simsetup.parse_time_params(time_dict)
     nt.assert_true(np.allclose(ttest['time_array'], times))
 
+    # check Ntimes = 1 and Nfreqs = 1 case
+    times = np.linspace(2458570, 2458570.5, 1)
+    tdict = pyuvsim.simsetup.time_array_to_params(times)
```

Comment:
Why is this in the `test_freq_parser` test? Shouldn't it be in `test_time_parser`?